src/test/java/liuni/LiUniResourceTests.java

```diff
@@ -0,0 +1,344 @@
+package liuni;
+
+import liuni.api.ErrorModel;
+import liuni.resources.LiUniResource;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import twitter4j.ResponseList;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import javax.ws.rs.core.Response;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+public class LiUniResourceTests {
+    @Mock private Twitter twitter;
+    @Mock private Status status;
+
+    @InjectMocks TwitterStatus twitterStatus;
+    @InjectMocks TwitterTimeline twitterTimeline;
+    @InjectMocks LiUniResource resource;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    /* Test LiUniResource .fetchTimeline() REST method */
+    @Test
+    public void testREST_fetchTimeline_emptyTimeline_Status() {
+        resource = new LiUniResource();
+        twitterTimeline = new TwitterTimeline();
+        twitterTimeline.setTwitter(twitter);
+        resource.setTwitterTimeline(twitterTimeline);
+
+        ResponseList<Status> respList = new ResponseListImpl<Status>();
+        boolean testResult;
+        try {
+            doReturn(respList).when(twitter).getHomeTimeline();
+
+            Response resp = resource.fetchTimeline();
+            testResult = resp.getStatus() == Response.Status.OK.getStatusCode();
```

Comment:
please verify the data payload of the response is expected as well and not just the status code